pkg/search/proxy/controller_test.go

```diff
@@ -418,7 +418,7 @@ func TestController_Connect(t *testing.T) {
 			conn.ServeHTTP(recorder, req)
 
 			response := recorder.Result()
-			fmt.Printf("response: %v", response)
+			t.Logf("response: %v\n", response)
```

Comment:
Why log this out?

Reply:
In addition, we don't need `\n` in `t.Logf`, since `A final newline is added if not provided.`

Reply:
Eh, you are right, delete it is better.